unicodetools/src/main/java/org/unicode/tools/Segmenter.java

```diff
@@ -1311,8 +1311,8 @@ public static void main(String[] args) throws IOException {
                             + "<!DOCTYPE ldml SYSTEM \"../../common/dtd/ldml.dtd\">\n"
                             + "<!--\n"
                             + "Copyright © 1991-2015 Unicode, Inc.\n"
```

Comment:
^ have a 2015 here

Reply:
changed to use `Default.getYear()` like elsewhere in these tools